Source/WebCore/Modules/fetch/FetchHeaders.h

```diff
@@ -52,22 +52,35 @@ class FetchHeaders : public RefCounted<FetchHeaders> {
     using Init = std::variant<Vector<Vector<String>>, Vector<KeyValuePair<String, String>>>;
     static ExceptionOr<Ref<FetchHeaders>> create(std::optional<Init>&&);
 
-    static Ref<FetchHeaders> create(Guard guard = Guard::None, HTTPHeaderMap&& headers = { }) { return adoptRef(*new FetchHeaders { guard, WTFMove(headers) }); }
+    static Ref<FetchHeaders> create(Guard guard = Guard::None, HTTPHeaderMap&& headers = { }, Vector<String>&& setCookieValues = { }) { return adoptRef(*new FetchHeaders { guard, WTFMove(headers), WTFMove(setCookieValues) }); }
     static Ref<FetchHeaders> create(const FetchHeaders& headers) { return adoptRef(*new FetchHeaders { headers }); }
 
     ExceptionOr<void> append(const String& name, const String& value);
     ExceptionOr<void> remove(const String&);
     ExceptionOr<String> get(const String&) const;
+    Vector<String> getSetCookie() const;
     ExceptionOr<bool> has(const String&) const;
     ExceptionOr<void> set(const String& name, const String& value);
 
     ExceptionOr<void> fill(const Init&);
     ExceptionOr<void> fill(const FetchHeaders&);
     void filterAndFill(const HTTPHeaderMap&, Guard);
 
-    String fastGet(HTTPHeaderName name) const { return m_headers.get(name); }
-    bool fastHas(HTTPHeaderName name) const { return m_headers.contains(name); }
-    void fastSet(HTTPHeaderName name, const String& value) { m_headers.set(name, value); }
+    String fastGet(HTTPHeaderName name) const
+    {
+        ASSERT(name != HTTPHeaderName::SetCookie);
+        return m_headers.get(name);
+    }
+    bool fastHas(HTTPHeaderName name) const
+    {
+        ASSERT(name != HTTPHeaderName::SetCookie);
+        return m_headers.contains(name);
+    }
+    void fastSet(HTTPHeaderName name, const String& value)
+    {
+        ASSERT(name != HTTPHeaderName::SetCookie);
+        m_headers.set(name, value);
+    }
```

Comment:
Since they are multilines, we can keep them in the header file but after the class declaration:
inline String FetchHeaders::fastGet(..._ {
}